stable/bookstack/Chart.yaml

```diff
@@ -2,11 +2,13 @@ apiVersion: v1
 appVersion: 0.24.3
 description: BookStack is a simple, self-hosted, easy-to-use platform for organising and storing information.
 name: bookstack
-version: 1.0.2
+version: 1.1.2
 home: https://www.bookstackapp.com/
 icon: https://github.com/BookStackApp/website/blob/master/static/images/logo.png
 sources:
 - https://github.com/BookStackApp/BookStack
 maintainers:
 - name: verwilst
   email: bart@verwilst.be
+- name: Snics
```

Comment:
did you sync up with the actual maintainer?

Reply:
I'm more than willing to add people to the maintainers list, but would prefer to only do it after prolonged and multiple fixes/additions. If we'd add everybody who does a commit, it would be quite the list, without actually representing the real situation.

Reply:
Ok, yeah, sure I can understand that I'm going to remove this. But would you like to continue working on this chart if it is ok for you?